src/Microsoft.DotNet.Darc/src/DarcLib.AzDev/AzureDevOpsClient.cs

```diff
@@ -651,7 +651,8 @@ public async Task<IList<Check>> GetPullRequestChecksAsync(string pullRequestUrl)
                 accountName,
                 projectName,
                 statusesPath,
-                _logger);
+                _logger,
+                versionOverride: "5.0-preview.1");
```

Comment:
@riarenas this was the reason it failed before :/

Reply:
This call is still using preview 5.0

Reply:
Do we have tests for this method / API? If not, can we improve the tests to catch this error at PR time instead of official build time?

Reply:
Scenario test runs only in local and staging builds not in PR builds. We have tests that cover this scenario.

Reply:
We can't really have local tests that hit AzDO... that would be the definition of not local...

Reply:
It would be very useful to be able to verify this integration sooner than after merge to master.

Reply:
You can run all but one of the tests against your dev environment. It's not feasible right now to automatically run them for you during PRs

Reply:
@epananth the issue this is trying to solve is that the 5.0 preview api versions might go away soon. What is the difference for this api between preview and release? I think we should fix our code to use the released api, otherwise we're still leaving a ticking time bomb that might blow up without warning. Things will just start to fail.

Reply:
I also think it's acceptable to move this API call to 5.1-preview.1 instead.

Reply:
I can't run the clone tests or the arcade-update tests.  A big problem is that they rarely pass and when they fail, there is next to no diagnosable information.  Even this one is hard to deal with, since the service throw HttpRequestExceptions all over the place as a normal course of operation... noticing that one of them has the message "400 Bad Arguments" instead of "404 Not Found" is an exercise in frustration.  So much frustration that I usually don't bother trying.